src/y2start/y2start

```diff
@@ -49,7 +49,9 @@ hostname = " @ #{hostname}" unless hostname.empty?
 
 title = "YaST2 - #{args[:client_name]}#{hostname}"
 # set title only if it is not CLI (bsc#1033993)
-Yast::UI.SetApplicationTitle(title) if args[:client_options][:params].empty?
+# modules do not have CLI arguments, except installation (bsc#1037891)
+set_title = args[:client_options][:params].empty? || args[:client_name] == "installation"
```

Comment:
So I guess that `args[:client_options]` and `args[:client_options][:params]` could not be `nil`, right?